src/client/messages.js

```diff
@@ -1,5 +1,8 @@
 export default {
   en: {
+    app: {
```

Comment:
BTW: About the order of keys in Maps - as I understand it: Map.prototype.values() can turn Map to Iterator, which has the keys in map ordered by the insertion order. That means that keys in Map have insertion index on which we can rely. But I'm not sure about the way reading messages.js to Map, if it's good idea to rely on the order of keys. Please advise. :-)